Dockerfile

```diff
@@ -39,4 +41,4 @@ WORKDIR /app
 COPY . ./
 COPY --from=build /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.9/site-packages
 
-ENTRYPOINT ["bash", "start.sh"]
\ No newline at end of file
+ENTRYPOINT ["/bin/bash", "-c", "sleep infinity"]
```

Comment:
why did you change an `entrypoint`?

Reply:
Forgot to revert. That was temporarily, to start "start.sh" straight from docker container. Reverted it